test/testsupport/testdata/Scenario.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package testsupport.testdata
+
+import timetopaytaxpayer.cor.model.SaUtr
+import uk.gov.hmrc.auth.core.{ConfidenceLevel, Enrolment}
+
+case class Scenario(
```

Comment:
I think it's better to define this class in the spec. The `Scenario` is pretty wide name a class with such name won't fit all specs. Better is to have such classes private in the specs. Sometimes might happen that they will have the same parameters but this is not a good reason to extract it and make it public to other code.

Reply:
makes sense, thanks!